projects/traddify/index.js

```diff
@@ -0,0 +1,14 @@
+const { sumTokens2, nullAddress } = require('../helper/unwrapLPs')
+
+const contract = "0xA7f3d2dEa7a53E7A9FEbBdE5Cf7C69d39D065030"
+
+async function tvl(time, ethBlock, { kava: block }) {
+  return sumTokens2({ tokens: [nullAddress], owner: contract, block, chain: 'kava', })

```

Comment:
change nullAddress to `"0xc86c7C0eFbd6A49B35E8714C5f59D99De09A225b" ` if you want to include kava